src/ResourceManager/ServiceFabric/AzureRM.ServiceFabric.psd1

```diff
@@ -75,13 +75,13 @@ FunctionsToExport = @()
 CmdletsToExport = 'Add-AzureRmServiceFabricApplicationCertificate', 
                'Add-AzureRmServiceFabricClientCertificate', 
                'Add-AzureRmServiceFabricClusterCertificate', 
-               'Add-AzureRmServiceFabricNode', 'Add-AzureRmServiceFabricNodeType', 
+               'Add-AzureRmServiceFabricNode', 
+               'Add-AzureRmServiceFabricNodeType', 
                'Get-AzureRmServiceFabricCluster', 
                'New-AzureRmServiceFabricCluster', 
                'Remove-AzureRmServiceFabricClientCertificate', 
                'Remove-AzureRmServiceFabricClusterCertificate', 
                'Remove-AzureRmServiceFabricNode', 
-               'Remove-AzureRmServiceFabricNodeType', 
```

Comment:
@QingChenmsft add a warning or error that gives them enough information not to use this cmdlet.